kafka-dockerized/scripts/kafka-brokers.sh

```diff
@@ -25,9 +25,10 @@ run_broker(){
   # shellcheck disable=SC2155
   local is_kafka_container="$(is_kafka_broker)"
   if [[ "$is_kafka_container" == 1 ]]; then
-    docker run -d --rm \
```

Comment:
As we have a script that removes all the containers, why don't just remove all "--rm" from each "running"?